xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-sources/pom.xml

```diff
@@ -31,7 +31,7 @@
   <name>XWiki Platform - Notifications - Sources</name>
   <description>Handle how notifications are retrieved</description>
   <properties>
-    <xwiki.jacoco.instructionRatio>0.78</xwiki.jacoco.instructionRatio>
+    <xwiki.jacoco.instructionRatio>0.71</xwiki.jacoco.instructionRatio>
```

Comment:
Why did you remove `DefaultParametrizedNotificationManagerTest` in this module?

Reply:
Because despite its very generic name, all the tests were actually relying on the old store. Did not had time to rewrite it entirely, so I just moved it.

Reply:
> so I just moved it.

you mean removed it? Or did I miss the new test?